bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/A5_09/A5_09_0C.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) 2010-2023 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.enocean.internal.eep.A5_09;
+
+import static org.openhab.binding.enocean.internal.EnOceanBindingConstants.*;
+
+import java.util.function.Function;
+
+import org.openhab.binding.enocean.internal.messages.ERP1Message;
+import org.openhab.core.config.core.Configuration;
+import org.openhab.core.library.types.QuantityType;
+import org.openhab.core.library.unit.Units;
+import org.openhab.core.types.State;
+
+/**
+ *
+ * @author Zhivka Dimova - Initial contribution
+ */
+public class A5_09_0C extends A5_09_05 {
+
+    public A5_09_0C(ERP1Message packet) {
+        super(packet);
+    }
+
+    protected String[] VOC_Indentifications = new String[] { "VOCT", "Formaldehyde", "Benzene", "Styrene", "Toluene",
```

Comment:
I would see this as bad design. If you really want to override the values of this array, I would suggest to add a new overrideable function, which returns this array.

Reply:
Thank you @fruggy83 for your suggestion - I changed it.